docs/aws/README.md

```diff
@@ -11,36 +13,61 @@ export AWS_DEFAULT_OUTPUT=json
 ```
 
 ### Create a Cluster
-Note: If you already have a cluster with version 1.19 or above, you may need to manually tag your subnets for Karpenter to work [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
 
-If your cluster version is 1.18 or below, you can skip this step.
+Create an EKS cluster
 ```bash
 eksctl create cluster \
 --name ${CLUSTER_NAME} \
---version 1.18 \
---region ${AWS_DEFAULT_REGION} \
 --node-type m5.large \
 --nodes 1 \
 --nodes-min 1 \
 --nodes-max 10 \
---managed
+--managed \
+--with-oidc
 ```
 
-### Setup IRSA, Karpenter Controller Role, and Karpenter Node Role
-We recommend using [CloudFormation](https://aws.amazon.com/cloudformation/) and [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) (IRSA) to manage these permissions. For production use, please review and restrict these permissions for your use case.
+Tag the cluster subnets with the required tags for Karpenter auto discovery.
+
+Note: If you already have a cluster with version 1.19 or below you can skip this step.
+More [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
+
+```bash
+export SUBNET_IDS=$(aws cloudformation describe-stacks \
+    --stack-name eksctl-${CLUSTER_NAME}-cluster \
+    --query 'Stacks[].Outputs[?OutputKey==`SubnetsPrivate`].OutputValue' \
+    --output text)
+
+aws ec2 create-tags \
+    --resources $(echo $SUBNET_IDS | tr ',' '\n') \
+    --tags Key="kubernetes.io/cluster/${CLUSTER_NAME}",Value=
+```
+
+For existing clusters you may need to add an OIDC provider for the cluster.
+We already added one with the `eksctl create cluster` command but it is safe to run again.
+
 ```bash
-# Enables IRSA for your cluster. This command is idempotent, but only needs to be executed once per cluster.
 eksctl utils associate-iam-oidc-provider \
---region ${AWS_DEFAULT_REGION} \
 --cluster ${CLUSTER_NAME} \
 --approve
+```
+
+### Setup IRSA, Karpenter Controller Role, and Karpenter Node Role
+We recommend using [CloudFormation](https://aws.amazon.com/cloudformation/) and [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) (IRSA) to manage these permissions.
+For production use, please review and restrict these permissions for your use case.
+
+```bash
+export OIDC_PROVIDER=$(aws eks describe-cluster \
+    --name ${CLUSTER_NAME} \
+    --query 'cluster.identity.oidc.issuer' \
+    --output text \
+    | sed 's,https://,,')
 
 # Creates IAM resources used by Karpenter
 aws cloudformation deploy \
   --stack-name Karpenter-${CLUSTER_NAME} \
-  --template-file ./docs/aws/karpenter.cloudformation.yaml \
+  --template-file  $(git rev-parse --show-toplevel)/docs/aws/karpenter.cloudformation.yaml \
```

Comment:
@njtran has an alternative fix for this one https://github.com/awslabs/karpenter/pull/470

Reply:
My fix would curl a release, while this fix would use the local file. For those who are installing it without cloning the repo, I think using curl makes a little more sense. WDYT @rothgar ?

Reply:
I thought about this some more and like your solution better. With a curl option we never require them to clone the repo. Only question for your option is if there's automation to update the release version when we make a new tag. Otherwise we'd have to always remember to update doc versions whenever we publish a new tag.

Reply:
Funny that you say that, Ellis and I talked about that yesterday, and I'll also be adding that to my PR :D